ios/FluentUI/Notification/NotificationTokens.swift

```diff
@@ -0,0 +1,182 @@
+    /// Defines the style of the notification.
+    public internal(set) var style: MSFNotificationStyle = .primaryToast
+
+    // MARK: - Design Tokens
+
+    /// The background color of the notification
+    open var backgroundColor: DynamicColor {
+        switch style {
+        case .primaryToast:
+            return globalTokens.brandColors[.tint40]
+        case .neutralToast:
+            return DynamicColor(light: ColorValue(0xF7F7F7), dark: ColorValue(0x393939))
+        case .primaryBar:
+            return DynamicColor(light: globalTokens.brandColors[.tint40].light, dark: globalTokens.brandColors[.tint10].dark)
+        case .primaryOutlineBar:
+            return DynamicColor(light: ColorValue(0xFFFFFF), dark: ColorValue(0x393939))
+        case .neutralBar:
+            return DynamicColor(light: ColorValue(0xDFDFDF), dark: ColorValue(0x393939))
+        case .dangerToast:
+            return DynamicColor(light: ColorValue(0xFDF6F6), dark: ColorValue(0x3F1011))
+        case .warningToast:
+            return DynamicColor(light: ColorValue(0xFFFBD6), dark: ColorValue(0x4C4400))
+        }
+    }
+
+    /// The color of the notification's foreground elements like text and icons
+    open var foregroundColor: DynamicColor {
+        switch style {
+        case .primaryToast:
+            return DynamicColor(light: globalTokens.brandColors[.shade10].light, dark: globalTokens.brandColors[.shade30].dark)
+        case .neutralToast:
+            return DynamicColor(light: ColorValue(0x393939), dark: ColorValue(0xF7F7F7))
+        case .primaryBar:
+            return DynamicColor(light: globalTokens.brandColors[.shade20].light, dark: ColorValue(0x000000))
+        case .primaryOutlineBar:
+            return DynamicColor(light: globalTokens.brandColors[.primary].light, dark: ColorValue(0xF7F7F7))
+        case .neutralBar:
+            return DynamicColor(light: ColorValue(0x090909), dark: ColorValue(0xF7F7F7))
+        case .dangerToast:
+            return DynamicColor(light: ColorValue(0xBC2F34), dark: ColorValue(0xDC5F63))
+        case .warningToast:
+            return DynamicColor(light: ColorValue(0x4C4400), dark: ColorValue(0xFDEA3D))
+        }
+    }
+
+    /// The value for the corner radius of the frame of the notification
+    open var cornerRadius: CGFloat {
+        switch style.isToast {
+        case true:
+            return globalTokens.borderRadius[.xLarge]
+        case false:
+            return globalTokens.borderSize[.none]
+        }
+    }
+
+    /// The value for the presentation offset of the notification
+    open var presentationOffset: CGFloat {
+        switch style.isToast {
+        case true:
+            return globalTokens.spacing[.medium]
+        case false:
+            return globalTokens.spacing[.none]
+        }
+    }
+
+    /// The value for the horizontal padding between the elements within a notification and its frame
+    open var horizontalPadding: CGFloat = 19.0
```

Comment:
Ah sorry just caught these: all tokens have to be generated properties so they can be properly overridden. So this line should read
```swift
open var horizontalPadding: CGFloat { 19.0 }
```

Same for the values below.